cmd/baton-jira/config.go

```diff
@@ -1,38 +1,17 @@
 package main
 
 import (
-	"context"
-	"errors"
-
-	"github.com/conductorone/baton-sdk/pkg/cli"
-	"github.com/spf13/cobra"
+	"github.com/conductorone/baton-sdk/pkg/field"
 )
 
-// config defines the external configuration required for the connector to run.
-type config struct {
-	cli.BaseConfig `mapstructure:",squash"` // Puts the base config options in the same place as the connector options
-
-	JiraUrl  string `mapstructure:"jira-url"`
-	Email    string `mapstructure:"jira-email"`
-	ApiToken string `mapstructure:"jira-api-token"`
-}
-
-// validateConfig is run after the configuration is loaded, and should return an error if it isn't valid.
-func validateConfig(ctx context.Context, cfg *config) error {
-	if cfg.Email == "" || cfg.ApiToken == "" {
-		return errors.New("jira-connector: BasicAuth options must be provided")
-	}
-
-	if cfg.JiraUrl == "" {
-		return errors.New("jira-connector: jira-url must be provided")
-	}
-
-	return nil
-}
+var (
+	jiraUrlField  = field.StringField("jira-url", field.WithRequired(true), field.WithDescription("Url to Jira service. ($BATON_JIRA_URL)"))
+	emailField    = field.StringField("jira-email", field.WithRequired(true), field.WithDescription("Email for Jira service. ($BATON_JIRA_EMAIL)"))
+	apiTokenField = field.StringField("jira-api-token", field.WithRequired(true), field.WithDescription("API token for Jira service. ($BATON_JIRA_API_TOKEN)"))
```

Comment:
oh you dont need to include the env var names anymore

Reply:
https://github.com/ConductorOne/baton-jira/pull/13/commits/a85332ace2698a3f3920d8084daf78f5a9941250